docs/content/config.md

```diff
@@ -178,16 +178,16 @@ actionMenu.DefaultMenu.AddMenu("open a menu", () => CreateAMenu(container, actio
 actionMenu.DefaultMenu.AddFreeForm("write to console", (s) => Console.WriteLine(s));
 ```
 
-note that `DefaultMenu` contains a useful set of default items. if you don't want these defaults, or you want to create an menu for nesting, you can use the `Clear` method. you can nest these menus as much as desired, so any set of menus can be created.
+Note that `DefaultMenu` contains a useful set of default items. If you don't want these defaults, or you want to create an menu for nesting, you can use the `Clear` method. You can nest these menus as much as desired, so any set of menus can be created.
 
 
-### **Gaps**
+### Gaps
 
-Gaps are supported through ``IConfigContext::AddLayoutProxy`` which is also used to recalculate window positions when the menu bar is present
+Gaps are supported through ``IConfigContext::AddLayoutProxy`` which is also used to recalculate window positions when the menu bar is present.
 
-in workspacer.config.csx
```

Comment:
I removed this because it's already known that we are in this file as said in the [quickstart](https://workspacer.org/quickstart/#configuring)